src/stko/_internal/calculators/extractors/xtb_extractor.py

```diff
@@ -152,6 +156,7 @@ def _properties_dict(self) -> dict[str, str]:
             "total_free_energy": "          | TOTAL FREE ENERGY  ",
             "ionisation_potential": "delta SCC IP (eV)",
             "electron_affinity": "delta SCC EA (eV)",
+            "total_sasa": "total SASA /",
```

Comment:
why the "/" at the end ?

Reply:
That is part of the string matching. Just the format of the output file.